homeassistant/components/ecowitt/sensor.py

```diff
@@ -196,6 +196,11 @@
         native_unit_of_measurement=PRESSURE_INHG,
         state_class=SensorStateClass.MEASUREMENT,
     ),
+    EcoWittSensorTypes.PERCENTAGE: SensorEntityDescription(
```

Comment:
This needs to be split out of this PR into a separate one.

Reply:
No time for that, but I removed the tag for next bugfix release